.travis.yml

```diff
@@ -19,6 +19,7 @@ matrix:
   - ghc: 8.4.4
   - ghc: 8.6.5
   - ghc: 8.8.1
```

Comment:
You probably also can update ghc-8.8.1 to ghc-8.8.3 :slightly_smiling_face:

Reply:
Good catch!